phpunit.xml

```diff
@@ -1,24 +1,24 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<phpunit bootstrap="bootstrap.php"
+<phpunit bootstrap="../../tests/bootstrap.php"
 		 strict="true"
 		 verbose="true"
 		 timeoutForSmallTests="900"
 		 timeoutForMediumTests="900"
 		 timeoutForLargeTests="900"
 		>
-	<testsuite name='ownCloud - Announcement Center App Tests'>
-		<directory suffix='Test.php'>.</directory>
+	<testsuite name='unit'>
+		<directory suffix='Test.php'>./tests</directory>
```

Comment:
The existing unit tests should be moved into ``tests/unit`` folder.
I guess we can try and do all this in a single PR!

Reply:
@phil-davis I tried moving them in unit folder but doing that breaks the unit tests. I think all the files namespace need changing we should try doing that in another PR